Library/Homebrew/requirements/php_requirement.rb

```diff
@@ -0,0 +1,11 @@
+class PhpRequirement < Requirement
+  fatal true
+  satisfy { which "php" }
+
+  env do
+    if which("php") != "/usr/bin/php"
```

Comment:
We don't want this; if you want to link against a custom `php`  you should make it first in your PATH.